ote/src/cljs/ote/style/form.cljs

```diff
@@ -9,6 +9,10 @@
 
 (def form-field {:margin-right "10px"})
 
+(def div-form-field {:padding-right "5px"
```

Comment:
rem should be preferred over pixel units, was there a specific problem?

Reply:
Good point. Taken